tests/core/entities/test_opossum.py

```diff
@@ -4,27 +4,27 @@
 import json
 from copy import deepcopy
 
-from opossum_lib.opossum.opossum_file_to_opossum_converter import (
-    OpossumFileToOpossumConverter,
+from opossum_lib.input_formats.opossum.services.convert_to_opossum import (  # noqa: E501
+    convert_to_opossum,
 )
-from tests.test_setup.opossum_faker_setup import OpossumFaker
+from tests.setup.opossum_faker_setup import OpossumFaker
 
 
-class TestOpossumModelToOpossumFileConversion:
+class TestOpossumToOpossumModelConversion:
     def test_moves_outfile(self, opossum_faker: OpossumFaker) -> None:
         opossum = opossum_faker.opossum()
 
-        result = opossum.to_opossum_file_format()
+        result = opossum.to_opossum_model()
 
         assert result.output_file == opossum.review_results
 
     def test_roundtrip(self, opossum_faker: OpossumFaker) -> None:
         opossum = opossum_faker.opossum()
         expected_result = deepcopy(opossum)
 
-        opossum_file = opossum.to_opossum_file_format()
+        opossum_file = opossum.to_opossum_model()
 
-        result = OpossumFileToOpossumConverter.convert_to_opossum(opossum_file)
+        result = convert_to_opossum(opossum_file)
 
         ## this can change due to the generation of new ids
         result_json = result.model_dump_json()
```

Comment:
I think instead of `model_dump_json` and then `json.loads` you could just do `model_dump` and directly get the `dict` or does that make a difference? Field aliases shouldn't play a role here I think (since we use the same parameters for the dumps, so there are either used or not)